src/metrics/hpr_metrics.erl

```diff
@@ -52,6 +76,15 @@ handle_cast(_Msg, State) ->
 
 handle_info(?METRICS_TICK, State) ->
     lager:debug("running metrics"),
+    _ = erlang:spawn_opt(
+        fun() ->
+            ok = record_grpc_connections()
+        end,
+        [
+            {fullsweep_after, 0},
```

Comment:
Do we need these options? Setting priority to high seems like premature optimization, unless you have prior experience that this is needed. Given that this process is short lived(?), do we really need to do a full sweep after minor GCs? This also feels like premature optimization.

Reply:
yeah might be much right now, we used to have it before in Router simple copy/paste here tbh